hardhat.config.js

```diff
@@ -7,7 +7,7 @@ require("@nomicfoundation/hardhat-chai-matchers")
  */
 module.exports = {
     solidity: {
-        version: '0.8.9',
+        version: '0.8.19',
```

Comment:
Solidity version needs to be updated, otherwise Compilation will be failed.

```
(using solidity version 0.8.9)
TypeError: Invalid contract specified in override list: "AxelarGMPExecutable".
  --> @axelar-network/axelar-gmp-sdk-solidity/contracts/express/AxelarExpressExecutable.sol:24:16:
   |
24 |     ) external override(AxelarGMPExecutable, IAxelarGMPExecutable) {
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Note: This contract:
  --> @axelar-network/axelar-gmp-sdk-solidity/contracts/executable/AxelarGMPExecutable.sol:13:1:
   |
13 | abstract contract AxelarGMPExecutable is IAxelarGMPExecutable {
   | ^ (Relevant source part starts here and spans across multiple lines).

Error HH600: Compilation failed
```

require to use `0.8.12` or above. 
There was a fix: `Inheritance: Consider functions in all ancestors during override analysis` in [solidity version 0.8.12. ](https://soliditylang.org/blog/2022/02/16/solidity-0.8.12-release-announcement/)

Reply:
The most recent version might introduce new dependencies or require code changes unrelated to the naming updates. However, it should use at least 0.8.12 for inheritance purposes. As you can see (https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/9847262542dd2279d3a14972f35cf85900657ca9/hardhat.config.js#L58), some contracts use fixed compiler settings to ensure deterministic bytecode across versions, which is why we're going with `0.8.19`. And if we want to upgrade the Solidity version to the latest, it's better to create a new PR for it.